packages/modules/viewer-react/src/services/iModel/ViewCreator3d.ts

```diff
@@ -0,0 +1,333 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Nit: double header

I know this will get replaced/removed when core is updated so probably not worth fixing

Reply:
fixed